From eb950ecf2cb0b11c893002f1b342a11ce74e3783 Mon Sep 17 00:00:00 2001
From: Fabian Wickborn <fabian@wickborn.net>
Date: Thu, 20 Aug 2015 10:45:51 +0200
Subject: [PATCH] build: Fix bootstrap.bash for official source tarballs

At the moment, bootstrap.bash assumes it is called from a git working
copy. Hence, it fails to complete when running in an unpacked official
source tarball where .git and .gitignore do not exist. This fix adds a
test for existence for .git and a -f switch for the removal of
.gitignore.

Fixes #12223

Change-Id: I7f305b83b38d5115504932bd38dadb7bdeb5d487
---

diff --git a/src/bootstrap.bash b/src/bootstrap.bash
index 60d6151..1b5ba7c 100755
--- a/src/bootstrap.bash
+++ b/src/bootstrap.bash
@@ -35,8 +35,10 @@
 cd "$targ"
 echo
 echo "#### Cleaning $targ"
-rm .gitignore
-git clean -f -d
+rm -f .gitignore
+if [ -e .git ]; then
+	git clean -f -d
+fi
 echo
 echo "#### Building $targ"
 echo
